### ISSUE TITLE
displayed blocksize is 0

### ISSUE THREAD

**Issue by giuliomoro** [pingdynasty/OwlControl#5]

**Comment by pingdynasty** [comment]
Before or after 'Load from OWL'?

**Comment by giuliomoro** [comment]
before :( would be good if it was to "load from owl" automatically when you open owlcontrol (and also refresh the list of midi devices every so often so it works if you connect the board after you launch owlcontrol)

**Comment by pingdynasty** [comment]
Yes it could be done automatically when the connection is established.
